File: freecodecamp/daily-code-challenge/2026-02-15.js
function getHillRating(drop, distance, type) {
  const steepness = drop / distance;
  let adjusted;

  if (type === "Downhill") {
    adjusted = steepness * 1.2;
  } else if (type === "Slalom") {
    adjusted = steepness * 0.9;
  } else if (type === "Giant Slalom") {
    adjusted = steepness * 1.0;
  } else {
    adjusted = steepness;
  }

  if (adjusted <= 0.1) {
    return "Green";
  } else if (adjusted <= 0.25) {
    return "Blue";
  } else {
    return "Black";
  }
}

File: freecodecamp/daily-code-challenge/2026-02-15.py
def get_hill_rating(drop, distance, hill_type):
    steepness = drop / distance
    
    if hill_type == "Downhill":
        adjusted = steepness * 1.2
    elif hill_type == "Slalom":
        adjusted = steepness * 0.9
    elif hill_type == "Giant Slalom":
        adjusted = steepness * 1.0
    else:
        adjusted = steepness
    
    if adjusted <= 0.1:
        return "Green"
    elif adjusted <= 0.25:
        return "Blue"
    else:
        return "Black"
